src/utils/schema/resolvers/Query/getFilter.ts

```diff
@@ -222,12 +227,24 @@ const buildMongoFilter = (
             break;
           case 'datetime':
           case 'datetime-local':
-            // startDatetime contains the beginning of the minute
-            startDatetime = getTimeForMongo(value);
-            // endDatetime contains the end of the minute (last second, last ms)
-            endDatetime = new Date(startDatetime.getTime() + 59999);
-            // we end up with a date range covering exactly the minute selected,
-            // regardless of the saved seconds and ms
+            //if we are using the {{today}} operator
+            if (
+              value === Placeholder.TODAY ||
+              REGEX_TODAY_MINUS.test(value) ||
```

Comment:
@MwanPygmay 
perhaps you should put that in a custom method, as such logic would also be needed in the other file called "getFilter"